src/exporter.js

```diff
@@ -105,7 +100,7 @@ function exporter (hash, dagService, options, callback) {
         rs.push(node.data)
         rs.push(null)
       }
-      ee.emit('file', {stream: rs, path: name})
+      this.push({stream: rs, path: name})
```

Comment:
If this function is the `dir exporter`, what kind of data needs to be pushed? A `dir` is a 'file' with no data/stream, just the path.